aodntools/timeseries_products/velocity_hourly_timeseries_template.json

```diff
@@ -0,0 +1,218 @@
+            "ancillary_variables": "VCUR_max, VCUR_min, VCUR_std, VCUR_count",
+            "long_name": "northward_sea_water_velocity",
+            "standard_name": "northward_sea_water_velocity",
+            "units": "m s-1",
+            "valid_max": 10.0,
+            "valid_min": -10.0
+            },
+        "VCUR_max": {
+            "units": "m s-1",
+            "standard_name": "northward_sea_water_velocity",
+            "long_name": "max data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:max"
+            },
+        "VCUR_min": {
+            "units": "m s-1",
+            "standard_name": "northward_sea_water_velocity",
+            "long_name": "min data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:min"
+            },
+        "VCUR_std": {
+            "units": "m s-1",
+            "standard_name": "northward_sea_water_velocity",
+            "long_name": "std data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:std"
+            },
+        "VCUR_count": {
+            "standard_name": "northward_sea_water_velocity number_of_observations",
+            "units": "1",
+            "long_name": "count data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:count"
+            },
+
+        "WCUR": {
+            "coordinates": "TIME DEPTH LATITUDE LONGITUDE",
+            "ancillary_variables": "WCUR_max, WCUR_min, WCUR_std, WCUR_count",
+            "long_name": "upward_sea_water_velocity",
+            "standard_name": "upward_sea_water_velocity",
+            "units": "m s-1",
+            "valid_max": 5.0,
+            "valid_min": -5.0
+            },
+        "WCUR_max": {
+            "units": "m s-1",
+            "standard_name": "upward_sea_water_velocity",
+            "long_name": "max data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:max"
+            },
+        "WCUR_min": {
+            "units": "m s-1",
+            "standard_name": "upward_sea_water_velocity",
+            "long_name": "min data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:min"
+            },
+        "WCUR_std": {
+            "units": "m s-1",
+            "standard_name": "upward_sea_water_velocity",
+            "long_name": "std data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:std"
+            },
+        "WCUR_count": {
+            "standard_name": "upward_sea_water_velocity number_of_observations",
+            "units": "1",
+            "long_name": "count data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:count"
+            },
+        "CELL_INDEX": {
+            "long_name": "index of the corresponding measuring cell",
+            "comment": "the closest cell to the sensor is cell 0"
```

Comment:
Add the same extended comment as in the aggregated product (https://github.com/aodn/python-aodntools/blob/master/aodntools/timeseries_products/velocity_aggregated_timeseries_template.json#L150)